glancesync/glancesync_config.py

```diff
@@ -187,6 +187,9 @@ def __init__(self, configuration_path=None, stream=None):
                 else:
                     target['only_tenant_images'] = True
 
+                target['obsolete_syncprops'] = configparser.getset(
+                        section, 'obsolete_syncprops')
```

Comment:
Add the new property **obsolete_syncprops** and its documentation to the template config file `conf/glancesync.conf`.
Perhaps it is interesting to have a boolean property to sync or not *obsolete* images...